global_planner/include/global_planner/planner_core.h

```diff
@@ -185,6 +186,7 @@ class GlobalPlanner : public nav_core::BaseGlobalPlanner {
         PotentialCalculator* p_calc_;
         Expander* planner_;
         Traceback* path_maker_;
+        OrientationFilter* o_filter_;
```

Comment:
This is only used in like 4 places -- so I would recommend making the name a bit more verbose (orientation_filter_) so people aren't having to look back and search for what the heck it is. (in the future, p_calc_ is another great variable name to update)